SU2_CFD/src/solver_structure.cpp

```diff
@@ -4825,6 +4825,365 @@ void CSolver::Read_InletFile_ASCII(CGeometry *geometry, CConfig *config, string
   
 }
 
+void CSolver::PrintNewInletData (passivedouble *Inlet_InterpolatedData, unsigned long nVertex,string Marker_Tag)
```

Comment:
Inlet_InterpolatedData can be `const passivedouble*`

Reply:
Right.